### PR TITLE
[MM-21572] Disable InstallFromUrl if EnableUploads is false

### PR DESCRIPTION
**Pull request by iomodo**
#### Summary
This PR will disable InsallFromUrl if EnableUploads is false

### CODE REVIEW

File: api4/plugin.go [hunk 1]
```diff
@@ -85,7 +85,7 @@ func uploadPlugin(c *Context, w http.ResponseWriter, r *http.Request) {
 }
 
 func installPluginFromUrl(c *Context, w http.ResponseWriter, r *http.Request) {
-	if !*c.App.Config().PluginSettings.Enable || *c.App.Config().PluginSettings.RequirePluginSignature {
+	if !*c.App.Config().PluginSettings.Enable || *c.App.Config().PluginSettings.RequirePluginSignature || !*c.App.Config().PluginSettings.EnableUploads {
```
Comment (agnivade) [inline api4/plugin.go]: Nit: let's break them into 3 separate lines.

**Comment by amyblais** [comment]
@iomodo Please help request a review from QA.

**Comment by prapti** [comment]
@iomodo The change works as expected and plugins cannot be uploaded from URL when EnableUploads is false, but there is also no server log that indicates why the plugin install did not work. Should some server log be added to avoid a confusion that the system is not broken when someone is trying to upload a plugin via URL when EnableUploads is false, esp. since trying to manually upload a plugin responds with this error: `Plugins and/or plugin uploads have been disabled.`

**Comment by prapti** [comment]
Also, is there a Jira ticket associated with this PR?

**Pull request merged**